dbms/src/Core/Settings.h

```diff
@@ -75,6 +75,7 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingUInt64, background_schedule_pool_size, 16, "Number of threads performing background tasks for replicated tables. Only has meaning at server startup.") \
     \
     M(SettingMilliseconds, distributed_directory_monitor_sleep_time_ms, 100, "Sleep time for StorageDistributed DirectoryMonitors in case there is no work or exception has been thrown.") \
+    M(SettingMilliseconds, distributed_directory_monitor_max_sleep_time_ms, 300000, "Maximum sleep time for StorageDistributed DirectoryMonitors, regardless occurred errors if any") \
```

Comment:
It's five minutes, not 30 seconds. What's the point of increase?

Reply:
My mistake, fixed